SwiftAA/Angles.swift

```diff
@@ -11,14 +11,34 @@ import Foundation
 /// The Degree is a unit of angle.
 /// Degree structs conform to SwiftAA Numeric type protocol.
 public struct Degree: NumericType, CustomStringConvertible {
-    /// The Degree value
-    public let value: Double
     
+    public var magnitude: Double
+    
+    public static func -=(lhs: inout Degree, rhs: Degree) {
+        lhs.value = lhs.value - rhs.value
+    }
+    
+    public static func +=(lhs: inout Degree, rhs: Degree) {
+        lhs.value = rhs.value + lhs.value
+    }
+    
+    public static func *(lhs: Degree, rhs: Degree) -> Degree {
+        return Degree(lhs.value * rhs.value)
+    }
```

Comment:
https://github.com/onekiloparsec/SwiftAA/pull/61#issuecomment-320232973

I believe this is what @onekiloparsec was referring to in his comment. These methods are all required for conformance of the protocol. We need to define how to get a new degree object when multiplying 2 other degree obj cats together.  I'm not sure how else we could do this. I had to do this for nearly every numeric type. I am open to suggestions on this.